### ISSUE TITLE
npm audit => 2 vulnerabilities (high & med)

### ISSUE THREAD

**Issue by dperez3** [postmanlabs/newman-reporter-html#92]
```bash
                      === npm audit security report ===                        
                                                                                
                                                                                
                                 Manual Review                                  
             Some vulnerabilities require your attention to resolve             
                                                                                
          Visit https://go.npm.me/audit-guide for additional guidance           
                                                                                
                                                                                
  High            Prototype Pollution                                           
                                                                                
  Package         handlebars                                                    
                                                                                
  Patched in      >=4.0.13                                                      
                                                                                
  Dependency of   newman-reporter-html [dev]                                    
                                                                                
  Path            newman-reporter-html > handlebars                             
                                                                                
  More info       https://npmjs.com/advisories/755                              
                                                                                
                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         lodash                                                        
                                                                                
  Patched in      >=4.17.11                                                     
                                                                                
  Dependency of   newman-reporter-html [dev]                                    
                                                                                
  Path            newman-reporter-html > lodash                                 
                                                                                
  More info       https://npmjs.com/advisories/782                              
                                                                                
found 2 vulnerabilities (1 moderate, 1 high) in 1025 scanned packages
  2 vulnerabilities require manual review. See the full report for details.
```

**Comment by codenirvana** [comment]
@dperez3 Thanks for reporting. 
Our `develop` branch already has the latest dependencies, will make a release from that.

**Comment by mlafleur** [comment]
Is there an ETA for this release?

**Comment by codenirvana** [comment]
Fixed in version `1.0.3`

**Comment by sup-engineer** [comment]
Not sure if this is fixed in 1.0.3 

Extract from my package-lock still shows newman-reporter-html v1.0.3 as using lodash 4.17.11
![image](https://user-images.githubusercontent.com/20064961/65036787-239ce880-d944-11e9-8eb1-d4a315f8b2e7.png)
